examples/log.c

```diff
@@ -63,7 +63,7 @@ struct log_options {
 static int parse_options(
 	struct log_state *s, struct log_options *opt, int argc, char **argv);
 static void print_time(const git_time *intime, const char *prefix);
-static void print_commit(git_commit *commit);
+static void print_commit(git_commit *commit, int show_log_size);
```

Comment:
It would feel more logical to me to pass a pointer to the  `log_options` struct here.